pandas/core/generic.py

```diff
@@ -7510,8 +7510,8 @@ def asfreq(
 
         Notes
         -----
-        To learn more about the frequency strings, please see `this link
-        <https://pandas.pydata.org/pandas-docs/stable/user_guide/timeseries.html#offset-aliases>`__.
+        To learn more about the frequency strings, please see this link:
+        :ref:`Timeseries <timeseries.offset_aliases>`
```

Comment:
This one should be kept as is (or with updated url if needed), as the docstrings are otherwise not useful as plain text

Reply:
(and the same for the other changes below in .py files)

Reply:
Ok, Reverting these changes